// @strapi/admin/build/6404.25d0f0d8.chunk.js
(self.webpackChunk_strapi_admin=self.webpackChunk_strapi_admin||[]).push([[6404],{6404:(we,Ae,Q)=>{"use strict";we.exports=Q(53490)},53490:function(we,Ae,Q){var je=Q(98399);(function(te,J){we.exports=J(Q(32735),Q(19615),Q(12788),Q(94009),Q(60672),Q(60530),Q(38938),Q(57235))})(this,function(te,J,H,R,W,X,ie,le){return function(o){var c={};function e(a){if(c[a])return c[a].exports;var i=c[a]={i:a,l:!1,exports:{}};return o[a].call(i.exports,i,i.exports,e),i.l=!0,i.exports}return e.m=o,e.c=c,e.d=function(a,i,f){e.o(a,i)||Object.defineProperty(a,i,{enumerable:!0,get:f})},e.r=function(a){typeof Symbol!="undefined"&&Symbol.toStringTag&&Object.defineProperty(a,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(a,"__esModule",{value:!0})},e.t=function(a,i){if(1&i&&(a=e(a)),8&i||4&i&&typeof a=="object"&&a&&a.__esModule)return a;var f=Object.create(null);if(e.r(f),Object.defineProperty(f,"default",{enumerable:!0,value:a}),2&i&&typeof a!="string")for(var l in a)e.d(f,l,function(t){return a[t]}.bind(null,l));return f},e.n=function(a){var i=a&&a.__esModule?function(){return a.default}:function(){return a};return e.d(i,"a",i),i},e.o=function(a,i){return Object.prototype.hasOwnProperty.call(a,i)},e.p="",e(e.s=95)}([function(o,c,e){o.exports=e(17)()},function(o,c){o.exports=te},function(o,c){o.exports=J},function(o,c){o.exports=function(e,a){return a||(a=e.slice(0)),Object.freeze(Object.defineProperties(e,{raw:{value:Object.freeze(a)}}))},o.exports.default=o.exports,o.exports.__esModule=!0},function(o,c,e){"use strict";e.r(c),e.d(c,"Box",function(){return p});var a,i=e(3),f=e.n(i),l=e(0),t=e.n(l),g=e(2),b=e.n(g),h=e(7),s={color:!0},p=b.a.div.withConfig({shouldForwardProp:function(n,r){return!s[n]&&r(n)}})(a||(a=f()([`
  // Font
  font-size: `,`;

  // Colors
  background: `,`;
  color: `,`;

  // Spaces
  `,`
  `,`
  `,`
  `,`
  `,`
  `,`
  `,`
  `,`
  `,`

  // Responsive hiding
  `,`
  `,`
  

  // Borders
  border-radius: `,`;
  border-style: `,`;
  border-width: `,`;
  border-color: `,`;
  border: `,`;

  // Shadows
  box-shadow: `,`;

  // Handlers
  pointer-events: `,`;
  &:hover {
    `,`
  }

  // Display
  display: `,`;

  // Position
  position: `,`;
  left: `,`;
  right: `,`;
  top: `,`;
  bottom: `,`;
  z-index: `,`;
  overflow: `,`;
  cursor: `,`;

  // Size
  width: `,`;
  max-width: `,`;
  min-width: `,`;
  height: `,`;
  max-height: `,`;
  min-height: `,`;

  // Animation
  transition: `,`;
  transform: `,`;
  animation: `,`;

  //Flexbox children props
  flex-shrink: `,`;
  flex-grow: `,`;
  flex-basis: `,`;
  flex: `,`;

  // Text
  text-align: `,`;
  text-transform: `,`;
  line-height: `,`;

  // Cursor
  cursor: `,`;
`])),function(n){var r=n.fontSize;return n.theme.fontSizes[r]||r},function(n){var r=n.theme,u=n.background;return r.colors[u]},function(n){var r=n.theme,u=n.color;return r.colors[u]},function(n){var r=n.theme,u=n.padding;return Object(h.a)("padding",u,r)},function(n){var r=n.theme,u=n.paddingTop;return Object(h.a)("padding-top",u,r)},function(n){var r=n.theme,u=n.paddingRight;return Object(h.a)("padding-right",u,r)},function(n){var r=n.theme,u=n.paddingBottom;return Object(h.a)("padding-bottom",u,r)},function(n){var r=n.theme,u=n.paddingLeft;return Object(h.a)("padding-left",u,r)},function(n){var r=n.theme,u=n.marginLeft;return Object(h.a)("margin-left",u,r)},function(n){var r=n.theme,u=n.marginRight;return Object(h.a)("margin-right",u,r)},function(n){var r=n.theme,u=n.marginTop;return Object(h.a)("margin-top",u,r)},function(n){var r=n.theme,u=n.marginBottom;return Object(h.a)("margin-bottom",u,r)},function(n){var r=n.theme;return n.hiddenS?"".concat(r.mediaQueries.tablet," { display: none; }"):void 0},function(n){var r=n.theme;return n.hiddenXS?"".concat(r.mediaQueries.mobile," { display: none; }"):void 0},function(n){var r=n.theme,u=n.hasRadius,E=n.borderRadius;return u?r.borderRadius:E},function(n){return n.borderStyle},function(n){return n.borderWidth},function(n){var r=n.borderColor;return n.theme.colors[r]},function(n){var r=n.theme,u=n.borderColor,E=n.borderStyle,z=n.borderWidth;if(u&&!E&&!z)return"1px solid ".concat(r.colors[u])},function(n){var r=n.theme,u=n.shadow;return r.shadows[u]},function(n){return n.pointerEvents},function(n){var r=n._hover,u=n.theme;return r?r(u):void 0},function(n){return n.display},function(n){return n.position},function(n){var r=n.left;return n.theme.spaces[r]||r},function(n){var r=n.right;return n.theme.spaces[r]||r},function(n){var r=n.top;return n.theme.spaces[r]||r},function(n){var r=n.bottom;return n.theme.spaces[r]||r},function(n){return n.zIndex},function(n){return n.overflow},function(n){return n.cursor},function(n){var r=n.width;return n.theme.spaces[r]||r},function(n){var r=n.maxWidth;return n.theme.spaces[r]||r},function(n){var r=n.minWidth;return n.theme.spaces[r]||r},function(n){var r=n.height;return n.theme.spaces[r]||r},function(n){var r=n.maxHeight;return n.theme.spaces[r]||r},function(n){var r=n.minHeight;return n.theme.spaces[r]||r},function(n){return n.transition},function(n){return n.transform},function(n){return n.animation},function(n){return n.shrink},function(n){return n.grow},function(n){return n.basis},function(n){return n.flex},function(n){return n.textAlign},function(n){return n.textTransform},function(n){return n.lineHeight},function(n){return n.cursor});p.displayName="Box",p.defaultProps={background:void 0,borderColor:void 0,color:void 0,hiddenS:!1,hiddenXS:!1,padding:void 0,paddingTop:void 0,paddingRight:void 0,paddingBottom:void 0,paddingLeft:void 0,hasRadius:!1,shadow:void 0,children:null,shrink:void 0,grow:void 0,basis:void 0,flex:void 0,_hover:function(){}},p.propTypes={_hover:t.a.func,background:t.a.string,basis:t.a.oneOfType([t.a.string,t.a.string]),borderColor:t.a.string,children:t.a.oneOfType([t.a.node,t.a.string]),color:t.a.string,flex:t.a.oneOfType([t.a.string,t.a.string]),grow:t.a.oneOfType([t.a.string,t.a.string]),hasRadius:t.a.bool,hiddenS:t.a.bool,hiddenXS:t.a.bool,padding:t.a.oneOfType([t.a.number,t.a.arrayOf(t.a.number)]),paddingBottom:t.a.oneOfType([t.a.number,t.a.arrayOf(t.a.number)]),paddingLeft:t.a.oneOfType([t.a.number,t.a.arrayOf(t.a.number)]),paddingRight:t.a.oneOfType([t.a.number,t.a.arrayOf(t.a.number)]),paddingTop:t.a.oneOfType([t.a.number,t.a.arrayOf(t.a.number)]),shadow:t.a.string,shrink:t.a.oneOfType([t.a.string,t.a.string])}},function(o,c,e){var a=e(22);o.exports=function(i,f){if(i==null)return{};var l,t,g=a(i,f);if(Object.getOwnPropertySymbols){var b=Object.getOwnPropertySymbols(i);for(t=0;t<b.length;t++)l=b[t],f.indexOf(l)>=0||Object.prototype.propertyIsEnumerable.call(i,l)&&(g[l]=i[l])}return g},o.exports.default=o.exports,o.exports.__esModule=!0},function(o,c){function e(){return o.exports=e=Object.assign||function(a){for(var i=1;i<arguments.length;i++){var f=arguments[i];for(var l in f)Object.prototype.hasOwnProperty.call(f,l)&&(a[l]=f[l])}return a},o.exports.default=o.exports,o.exports.__esModule=!0,e.apply(this,arguments)}o.exports=e,o.exports.default=o.exports,o.exports.__esModule=!0},function(o,c,e){"use strict";var a=e(10),i=e.n(a),f=e(13),l=e.n(f);c.a=function(t,g,b){var h=g;if(Array.isArray(g)||l()(g)!=="object"||(h=[g==null?void 0:g.desktop,g==null?void 0:g.tablet,g==null?void 0:g.mobile]),h!==void 0){if(Array.isArray(h)){var s=h,p=i()(s,3),n=p[0],r=p[1],u=p[2],E="".concat(t,": ").concat(b.spaces[n],";");return r!==void 0&&(E+="".concat(b.mediaQueries.tablet,`{
          `).concat(t,": ").concat(b.spaces[r],`;
        }`)),u!==void 0&&(E+="".concat(b.mediaQueries.mobile,`{
          `).concat(t,": ").concat(b.spaces[u],`;
        }`)),E}var z=b.spaces[h]||h;return"".concat(t,": ").concat(z,";")}}},function(o,c,e){"use strict";e.r(c),e.d(c,"Typography",function(){return s});var a,i=e(3),f=e.n(i),l=e(0),t=e.n(l),g=e(2),b=["alpha","beta","delta","epsilon","omega","pi","sigma"],h={fontSize:!0,fontWeight:!0},s=e.n(g).a.span.withConfig({shouldForwardProp:function(p,n){return!h[p]&&n(p)}})(a||(a=f()([`
  font-weight: `,`;
  font-size: `,`;
  line-height: `,`;
  color: `,`;
  text-transform: `,`;
  `,`
  `,`
`])),function(p){var n=p.theme,r=p.fontWeight;return n.fontWeights[r]},function(p){var n=p.theme,r=p.fontSize;return n.fontSizes[r]},function(p){var n=p.theme,r=p.lineHeight;return n.lineHeights[r]},function(p){var n=p.theme,r=p.textColor;return n.colors[r||"neutral800"]},function(p){return p.textTransform},function(p){return p.ellipsis&&`
    display: block;
    white-space: nowrap;
    overflow: hidden;
    text-overflow: ellipsis;
  `},function(p){var n=p.variant,r=p.theme;switch(n){case"alpha":return`
        font-weight: `.concat(r.fontWeights.bold,`;
        font-size: `).concat(r.fontSizes[5],`;
        line-height: `).concat(r.lineHeights[2],`;
      `);case"beta":return`
        font-weight: `.concat(r.fontWeights.bold,`;
        font-size: `).concat(r.fontSizes[4],`;
        line-height: `).concat(r.lineHeights[1],`;
      `);case"delta":return`
        font-weight: `.concat(r.fontWeights.semiBold,`;
        font-size: `).concat(r.fontSizes[3],`;
        line-height: `).concat(r.lineHeights[2],`;
      `);case"epsilon":return`
        font-size: `.concat(r.fontSizes[3],`;
        line-height: `).concat(r.lineHeights[6],`;
      `);case"omega":return`
        font-size: `.concat(r.fontSizes[2],`;
        line-height: `).concat(r.lineHeights[4],`;
      `);case"pi":return`
        font-size: `.concat(r.fontSizes[1],`;
        line-height: `).concat(r.lineHeights[3],`;
      `);case"sigma":return`
        font-weight: `.concat(r.fontWeights.bold,`;
        font-size: `).concat(r.fontSizes[0],`;
        line-height: `).concat(r.lineHeights[5],`;
        text-transform: uppercase;
      `);default:return`
        font-size: `.concat(r.fontSizes[2],`;
      `)}});s.defaultProps={fontWeight:void 0,fontSize:void 0,lineHeight:void 0,textColor:void 0,textTransform:void 0,variant:"omega"},s.propTypes={fontSize:t.a.oneOfType([t.a.number,t.a.string]),fontWeight:t.a.string,lineHeight:t.a.oneOfType([t.a.number,t.a.string]),textColor:t.a.string,textTransform:t.a.string,variant:t.a.oneOf(b)}},function(o,c,e){"use strict";e.r(c),e.d(c,"Flex",function(){return p});var a,i=e(3),f=e.n(i),l=e(0),t=e.n(l),g=e(2),b=e.n(g),h=e(4),s={direction:!0},p=b()(h.Box).withConfig({shouldForwardProp:function(n,r){return!s[n]&&r(n)}})(a||(a=f()([`
  display: `,`;
  flex-direction: `,`;
  justify-content: `,`;
  align-items: `,`;
  flex-wrap: `,`;
`])),function(n){return n.inline?"inline-flex":"flex"},function(n){return n.direction},function(n){return n.justifyContent},function(n){return n.alignItems},function(n){return n.wrap});p.defaultProps={alignItems:"center",basis:void 0,direction:"row",inline:!1,justifyContent:void 0,reverse:!1,wrap:void 0},p.propTypes={alignItems:t.a.string,basis:t.a.oneOfType([t.a.string,t.a.number]),direction:t.a.string,inline:t.a.bool,justifyContent:t.a.string,reverse:t.a.bool,wrap:t.a.string}},function(o,c,e){var a=e(23),i=e(24),f=e(20),l=e(25);o.exports=function(t,g){return a(t)||i(t,g)||f(t,g)||l()},o.exports.default=o.exports,o.exports.__esModule=!0},function(o,c,e){"use strict";e.d(c,"a",function(){return a});var a={DOWN:"ArrowDown",UP:"ArrowUp",RIGHT:"ArrowRight",LEFT:"ArrowLeft",ESCAPE:"Escape",ENTER:"Enter",SPACE:" ",TAB:"Tab",END:"End",HOME:"Home",DELETE:"Delete",PAGE_UP:"PageUp",PAGE_DOWN:"PageDown",BACKSPACE:"Backspace",CLEAR:"Clear"}},function(o,c,e){"use strict";e.d(c,"a",function(){return i}),e.d(c,"b",function(){return f});var a=e(1),i=Object(a.createContext)({error:void 0,hint:void 0,name:"",id:""}),f=function(){return Object(a.useContext)(i)}},function(o,c){function e(a){return typeof Symbol=="function"&&typeof Symbol.iterator=="symbol"?(o.exports=e=function(i){return typeof i},o.exports.default=o.exports,o.exports.__esModule=!0):(o.exports=e=function(i){return i&&typeof Symbol=="function"&&i.constructor===Symbol&&i!==Symbol.prototype?"symbol":typeof i},o.exports.default=o.exports,o.exports.__esModule=!0),e(a)}o.exports=e,o.exports.default=o.exports,o.exports.__esModule=!0},function(o,c,e){"use strict";e.d(c,"b",function(){return a}),e.d(c,"c",function(){return i}),e.d(c,"a",function(){return f});var a=function(l){return function(t){var g=t.theme,b=t.size;return g.sizes[l][b]}},i=function(){var l=arguments.length>0&&arguments[0]!==void 0?arguments[0]:"&";return function(t){var g=t.theme,b=t.hasError;return`
      outline: none;
      box-shadow: 0;
      transition-property: border-color, box-shadow, fill;
      transition-duration: 0.2s;

      `.concat(l,`:focus-within {
        border: 1px solid `).concat(b?g.colors.danger600:g.colors.primary600,`;
        box-shadow: `).concat(b?g.colors.danger600:g.colors.primary600,` 0px 0px 0px 2px;
      }
    `)}},f=function(l){var t=l.theme;return`
  position: relative;
  outline: none;
  
  &:after {
    transition-property: all;
    transition-duration: 0.2s;
    border-radius: 8px;
    content: '';
    position: absolute;
    top: -4px;
    bottom: -4px;
    left: -4px;
    right: -4px;
    border: 2px solid transparent;
  }

  &:focus-visible {
    outline: none;
    &:after {
      border-radius: 8px;
      content: '';
      position: absolute;
      top: -5px;
      bottom: -5px;
      left: -5px;
      right: -5px;
      border: 2px solid `.concat(t.colors.primary600,`;
    }
  }
`)}},function(o,c,e){"use strict";e.d(c,"a",function(){return f});var a=e(1),i=0,f=function(l,t){return Object(a.useRef)(t||"".concat(l,"-").concat(++i)).current}},function(o,c,e){"use strict";e.r(c),e.d(c,"Field",function(){return p}),e.d(c,"FieldLabel",function(){return d}),e.d(c,"InputWrapper",function(){return M.b}),e.d(c,"FieldInput",function(){return M.a}),e.d(c,"FieldHint",function(){return j}),e.d(c,"FieldError",function(){return U}),e.d(c,"FieldContext",function(){return b.a}),e.d(c,"useField",function(){return b.b}),e.d(c,"FieldAction",function(){return w});var a=e(5),i=e.n(a),f=e(1),l=e.n(f),t=e(0),g=e.n(t),b=e(12),h=e(15),s=["children","name","error","hint","id"],p=function(y){var P=y.children,S=y.name,x=y.error,N=y.hint,q=y.id,G=i()(y,s),Z=Object(h.a)("field",q);return l.a.createElement("div",G,l.a.createElement(b.a.Provider,{value:{name:S,id:Z,error:x,hint:N}},P))};p.defaultProps={error:void 0,hint:void 0,id:void 0,name:void 0},p.propTypes={children:g.a.node.isRequired,error:g.a.string,hint:g.a.string,id:g.a.oneOfType([g.a.string,g.a.number]),name:g.a.string};var n,r,u=e(6),E=e.n(u),z=e(3),A=e.n(z),F=e(2),O=e.n(F),L=e(9),I=e(8),k=["children","required","action"],m=O()(I.Typography)(n||(n=A()([`
  line-height: 0;
`]))),T=O()(L.Flex)(r||(r=A()([`
  line-height: 0;

  svg path {
    fill: `,`;
  }
`])),function(y){return y.theme.colors.neutral500}),d=function(y){var P=y.children,S=y.required,x=y.action,N=i()(y,k),q=Object(b.b)().id;return l.a.createElement(I.Typography,E()({variant:"pi",textColor:"neutral800",htmlFor:q,fontWeight:"bold",as:"label",required:S},N),l.a.createElement(L.Flex,{alignItems:"center"},P,S&&l.a.createElement(m,{textColor:"danger600"},"*"),x&&l.a.createElement(T,{marginLeft:1},x)))};d.defaultProps={required:!1,action:void 0},d.propTypes={action:g.a.element,children:g.a.node.isRequired,required:g.a.bool};var C,M=e(32),j=function(){var y=Object(b.b)(),P=y.id,S=y.hint,x=y.error;return!S||x?null:l.a.createElement(I.Typography,{variant:"pi",as:"p",id:"".concat(P,"-hint"),textColor:"neutral600"},S)},U=function(){var y=Object(b.b)(),P=y.id,S=y.error;return S?l.a.createElement(I.Typography,{variant:"pi",as:"p",id:"".concat(P,"-error"),textColor:"danger600","data-strapi-field-error":!0},S):null},B=["label","children"],D=O.a.button(C||(C=A()([`
  border: none;
  background: transparent;
  // TODO: Make sure to use the theme when it's ready
  font-size: 1.6rem;
  width: auto;
  padding: 0;
  display: flex;
  align-items: center;
`]))),w=function(y){var P=y.label,S=y.children,x=i()(y,B);return l.a.createElement(D,E()({"aria-label":P,type:"button"},x),S)};w.propTypes={children:g.a.node.isRequired,label:g.a.string.isRequired}},function(o,c,e){"use strict";var a=e(18);function i(){}function f(){}f.resetWarningCache=i,o.exports=function(){function l(b,h,s,p,n,r){if(r!==a){var u=new Error("Calling PropTypes validators directly is not supported by the `prop-types` package. Use PropTypes.checkPropTypes() to call them. Read more at http://fb.me/use-check-prop-types");throw u.name="Invariant Violation",u}}function t(){return l}l.isRequired=l;var g={array:l,bool:l,func:l,number:l,object:l,string:l,symbol:l,any:l,arrayOf:t,element:l,elementType:l,instanceOf:t,node:l,objectOf:t,oneOf:t,oneOfType:t,shape:t,exact:t,checkPropTypes:f,resetWarningCache:i};return g.PropTypes=g,g}},function(o,c,e){"use strict";o.exports="SECRET_DO_NOT_PASS_THIS_OR_YOU_WILL_BE_FIRED"},function(o,c){o.exports=function(e,a){(a==null||a>e.length)&&(a=e.length);for(var i=0,f=new Array(a);i<a;i++)f[i]=e[i];return f},o.exports.default=o.exports,o.exports.__esModule=!0},function(o,c,e){var a=e(19);o.exports=function(i,f){if(i){if(typeof i=="string")return a(i,f);var l=Object.prototype.toString.call(i).slice(8,-1);return l==="Object"&&i.constructor&&(l=i.constructor.name),l==="Map"||l==="Set"?Array.from(i):l==="Arguments"||/^(?:Ui|I)nt(?:8|16|32)(?:Clamped)?Array$/.test(l)?a(i,f):void 0}},o.exports.default=o.exports,o.exports.__esModule=!0},function(o,c,e){"use strict";e.r(c),e.d(c,"VisuallyHidden",function(){return t});var a,i=e(3),f=e.n(i),l=e(2),t=e.n(l).a.div(a||(a=f()([`
  border: 0;
  clip: rect(0 0 0 0);
  height: 1px;
  margin: -1px;
  overflow: hidden;
  padding: 0;
  position: absolute;
  width: 1px;
`])))},function(o,c){o.exports=function(e,a){if(e==null)return{};var i,f,l={},t=Object.keys(e);for(f=0;f<t.length;f++)i=t[f],a.indexOf(i)>=0||(l[i]=e[i]);return l},o.exports.default=o.exports,o.exports.__esModule=!0},function(o,c){o.exports=function(e){if(Array.isArray(e))return e},o.exports.default=o.exports,o.exports.__esModule=!0},function(o,c){o.exports=function(e,a){var i=e==null?null:typeof Symbol!="undefined"&&e[Symbol.iterator]||e["@@iterator"];if(i!=null){var f,l,t=[],g=!0,b=!1;try{for(i=i.call(e);!(g=(f=i.next()).done)&&(t.push(f.value),!a||t.length!==a);g=!0);}catch(h){b=!0,l=h}finally{try{g||i.return==null||i.return()}finally{if(b)throw l}}return t}},o.exports.default=o.exports,o.exports.__esModule=!0},function(o,c){o.exports=function(){throw new TypeError(`Invalid attempt to destructure non-iterable instance.
In order to be iterable, non-array objects must have a [Symbol.iterator]() method.`)},o.exports.default=o.exports,o.exports.__esModule=!0},function(o,c,e){"use strict";e.r(c),e.d(c,"Stack",function(){return k});var a,i,f=e(6),l=e.n(f),t=e(5),g=e.n(t),b=e(3),h=e.n(b),s=e(1),p=e.n(s),n=e(0),r=e.n(n),u=e(2),E=e.n(u),z=e(4),A=e(9),F=["horizontal","spacing","size"],O={size:!0},L=E()(z.Box).withConfig({shouldForwardProp:function(m,T){return!O[m]&&T(m)}})(a||(a=h()([`
  display: flex;
  flex-direction: column;

  & > * {
    margin-top: 0;
    margin-bottom: 0;
  }

  & > * + * {
    margin-top: `,`;
  }
`])),function(m){var T=m.theme,d=m.spacing;return T.spaces[d]}),I=E()(A.Flex).withConfig({shouldForwardProp:function(m,T){return!O[m]&&T(m)}})(i||(i=h()([`
  & > * {
    margin-left: 0;
    margin-right: 0;
  }

  & > * + * {
    margin-left: `,`;
  }
`])),function(m){var T=m.theme,d=m.spacing;return T.spaces[d]}),k=Object(s.forwardRef)(function(m,T){var d=m.horizontal,C=m.spacing,M=m.size,j=g()(m,F);return M&&je.warn('Deprecation warning: Usage of "size" prop in Stack component is deprecated. This is discouraged and will be removed in the next major release. Please use "spacing" instead'),d?p.a.createElement(I,l()({ref:T,spacing:C||M},j)):p.a.createElement(L,l()({ref:T,spacing:C||M},j))});k.displayName="Stack",k.defaultProps={horizontal:!1,size:void 0,spacing:void 0},k.propTypes={horizontal:r.a.bool,size:r.a.number,spacing:r.a.number}},function(o,c,e){"use strict";e.d(c,"a",function(){return a});var a={input:{S:"".concat(2,"rem"),M:"".concat(2.5,"rem")},accordions:{S:"".concat(3,"rem"),M:"".concat(5.5,"rem")}}},,function(o,c,e){"use strict";e.r(c),e.d(c,"Portal",function(){return t});var a=e(10),i=e.n(a),f=e(1),l=e(33),t=function(g){var b=g.children,h=Object(f.useRef)(null),s=Object(f.useState)(!1),p=i()(s,2),n=p[0],r=p[1];return Object(f.useLayoutEffect)(function(){return h.current=document.createElement("div"),h.current.setAttribute("data-react-portal","true"),document.body.appendChild(h.current),r(!0),function(){var u;(u=h.current)===null||u===void 0||u.remove()}},[]),n&&h.current?Object(l.createPortal)(b,h.current):null}},function(o,c){o.exports=function(e,a,i){return a in e?Object.defineProperty(e,a,{value:i,enumerable:!0,configurable:!0,writable:!0}):e[a]=i,e},o.exports.default=o.exports,o.exports.__esModule=!0},,function(o,c,e){"use strict";e.d(c,"b",function(){return m}),e.d(c,"a",function(){return T});var a,i,f=e(6),l=e.n(f),t=e(5),g=e.n(t),b=e(3),h=e.n(b),s=e(1),p=e.n(s),n=e(2),r=e.n(n),u=e(0),E=e.n(u),z=e(27),A=e(14),F=e(12),O=e(9),L=e(4),I=["endAction","startAction","disabled","onChange","size"],k=r.a.input(a||(a=h()([`
  border: none;
  border-radius: `,`;
  padding-left: `,`;
  padding-right: `,`;
  cursor: `,`;

  color: `,`;
  font-weight: 400;
  // TODO: Make sure to use the theme when it's ready
  font-size: `,`rem;
  display: block;
  width: 100%;
  background: inherit;

  ::placeholder {
    color: `,`;
    opacity: 1;
  }

  &[aria-disabled='true'] {
    color: inherit;
  }

  //focus managed by InputWrapper
  &:focus {
    outline: none;
    box-shadow: none;
  }
`])),function(d){return d.theme.borderRadius},function(d){var C=d.theme;return d.hasLeftAction?0:C.spaces[4]},function(d){var C=d.theme;return d.hasRightAction?0:C.spaces[4]},function(d){return d["aria-disabled"]?"not-allowed":void 0},function(d){return d.theme.colors.neutral800},.875,function(d){return d.theme.colors.neutral500}),m=r()(O.Flex)(i||(i=h()([`
  border: 1px solid `,`;
  border-radius: `,`;
  background: `,`;
  height: `,`;

  `,`

  `,`
`])),function(d){var C=d.theme;return d.hasError?C.colors.danger600:C.colors.neutral200},function(d){return d.theme.borderRadius},function(d){return d.theme.colors.neutral0},Object(A.b)("input"),Object(A.c)(),function(d){var C=d.theme;return d.disabled?`
    color: `.concat(C.colors.neutral600,`;
    background: `).concat(C.colors.neutral150,`;
  
  `):void 0}),T=Object(s.forwardRef)(function(d,C){var M,j=d.endAction,U=d.startAction,B=d.disabled,D=d.onChange,w=d.size,y=g()(d,I),P=Object(F.b)(),S=P.id,x=P.error,N=P.hint,q=P.name;x?M="".concat(S,"-error"):N&&(M="".concat(S,"-hint"));var G=Boolean(x);return p.a.createElement(m,{size:w,justifyContent:"space-between",hasError:G,disabled:B},U&&p.a.createElement(L.Box,{paddingLeft:3,paddingRight:2},U),p.a.createElement(k,l()({id:S,name:q,ref:C,"aria-describedby":M,"aria-invalid":G,"aria-disabled":B,hasLeftAction:Boolean(U),hasRightAction:Boolean(j),onChange:function(Z){B||D(Z)}},y)),j&&p.a.createElement(L.Box,{paddingLeft:2,paddingRight:3},j))});T.displayName="FieldInput",T.defaultProps={disabled:!1,endAction:void 0,size:"M",startAction:void 0,onChange:function(){}},T.propTypes={disabled:E.a.bool,endAction:E.a.element,onChange:E.a.func,size:E.a.oneOf(Object.keys(z.a.input)),startAction:E.a.element}},function(o,c){o.exports=H},function(o,c){o.exports=R},function(o,c){o.exports=W},function(o,c){o.exports=X},,,function(o,c,e){"use strict";e.r(c),e.d(c,"BaseButtonWrapper",function(){return A}),e.d(c,"BaseButton",function(){return F});var a,i=e(6),f=e.n(i),l=e(5),t=e.n(l),g=e(3),b=e.n(g),h=e(1),s=e.n(h),p=e(0),n=e.n(p),r=e(2),u=e.n(r),E=e(14),z=["disabled","children"],A=u.a.button(a||(a=b()([`
  display: flex;
  cursor: pointer;
  padding: `,`;
  border-radius: `,`;
  background: `,`;
  border: 1px solid `,`;
  svg {
    height: `,`;
    width: `,`;
  }
  svg {
    > g,
    path {
      fill: `,`;
    }
  }
  &[aria-disabled='true'] {
    pointer-events: none;
  }

  `,`
`])),function(O){return O.theme.spaces[2]},function(O){return O.theme.borderRadius},function(O){return O.theme.colors.neutral0},function(O){return O.theme.colors.neutral200},function(O){return O.theme.spaces[3]},function(O){return O.theme.spaces[3]},function(O){return O.theme.colors.neutral0},E.a),F=s.a.forwardRef(function(O,L){var I=O.disabled,k=O.children,m=t()(O,z);return s.a.createElement(A,f()({ref:L,"aria-disabled":I,type:"button",disabled:I},m),k)});F.displayName="BaseButton",F.defaultProps={disabled:!1},F.propTypes={children:n.a.node.isRequired,disabled:n.a.bool}},,,,,,,function(o,c,e){"use strict";e.r(c),e.d(c,"Tooltip",function(){return U});var a,i=e(6),f=e.n(i),l=e(30),t=e.n(l),g=e(5),b=e.n(g),h=e(3),s=e.n(h),p=e(1),n=e.n(p),r=e(0),u=e.n(r),E=e(2),z=e.n(E),A=e(4),F=e(8),O=e(29),L=e(10),I=e.n(L),k=function(B,D,w){var y=B.getBoundingClientRect(),P=D.getBoundingClientRect();return w==="bottom"?function(S,x){var N=(S.width-x.width)/2;return{left:x.left-N,top:x.top+x.height+8+window.pageYOffset}}(y,P):w==="right"?function(S,x){var N=(S.height-x.height)/2;return{left:x.left+x.width+8,top:x.top-N+window.pageYOffset}}(y,P):w==="left"?function(S,x){var N=(S.height-x.height)/2;return{left:x.left-S.width-8,top:x.top-N+window.pageYOffset}}(y,P):function(S,x){var N=(S.width-x.width)/2,q=x.left-N,G=x.top-S.height-8+window.pageYOffset,Z=window.innerWidth-x.right;return x.left+S.width-Z>window.innerWidth?(q=x.left-S.width-8,G=x.top+window.scrollY-x.height/2):q<0?(q=x.width+x.left+8,G=x.top+window.scrollY-S.height/2+8):G<0&&q>0&&(G=x.top+x.height+8),{left:q,top:G}}(y,P)},m=e(15),T=e(21),d=["children","label","description","delay","position","id"],C=["visible"];function M(B,D){var w=Object.keys(B);if(Object.getOwnPropertySymbols){var y=Object.getOwnPropertySymbols(B);D&&(y=y.filter(function(P){return Object.getOwnPropertyDescriptor(B,P).enumerable})),w.push.apply(w,y)}return w}var j=z()(A.Box)(a||(a=s()([`
  position: absolute;
  /* z-index exist because of its position inside Modals */
  z-index: 4;
  display: `,`;
`])),function(B){return B.visible?"revert":"none"}),U=function(B){var D=B.children,w=B.label,y=B.description,P=B.delay,S=B.position,x=B.id,N=b()(B,d),q=Object(m.a)("tooltip",x),G=Object(m.a)("description"),Z=function(ee){var ne=Object(p.useState)(!1),V=I()(ne,2),$=V[0],oe=V[1],de=Object(p.useRef)(),fe=function(){de.current&&clearTimeout(de.current)};return Object(p.useEffect)(function(){return function(){fe()}},[]),{visible:$,onFocus:function(){oe(!0)},onBlur:function(){oe(!1)},onMouseEnter:function(){de.current=setTimeout(function(){oe(!0)},ee)},onMouseLeave:function(){fe(),oe(!1)}}}(P),ue=Z.visible,Ee=b()(Z,C),se=function(ee,ne){var V=Object(p.useRef)(null),$=Object(p.useRef)(null);return Object(p.useLayoutEffect)(function(){if(ee){var oe=V.current,de=$.current,fe=k(oe,de,ne);oe.style.left="".concat(fe.left,"px"),oe.style.top="".concat(fe.top,"px")}},[ee]),{tooltipWrapperRef:V,toggleSourceRef:$}}(ue,S),Ce=se.tooltipWrapperRef,be=se.toggleSourceRef,Te=n.a.cloneElement(D,function(ee){for(var ne=1;ne<arguments.length;ne++){var V=arguments[ne]!=null?arguments[ne]:{};ne%2?M(Object(V),!0).forEach(function($){t()(ee,$,V[$])}):Object.getOwnPropertyDescriptors?Object.defineProperties(ee,Object.getOwnPropertyDescriptors(V)):M(Object(V)).forEach(function($){Object.defineProperty(ee,$,Object.getOwnPropertyDescriptor(V,$))})}return ee}({tabIndex:0,"aria-labelledby":w?q:void 0,"aria-describedby":y?q:void 0},Ee));return n.a.createElement(n.a.Fragment,null,n.a.createElement(O.Portal,null,n.a.createElement(j,f()({id:q,background:"neutral900",hasRadius:!0,padding:2,role:"tooltip",ref:Ce,visible:ue},N),ue&&n.a.createElement(T.VisuallyHidden,{id:G},y),n.a.createElement(F.Typography,{as:"p",variant:"pi",fontWeight:"bold",textColor:"neutral0"},w||y))),n.a.createElement("span",{ref:be},Te))};U.defaultProps={delay:500,id:void 0,position:"top",label:void 0,description:void 0},U.propTypes={children:u.a.node.isRequired,delay:u.a.number,description:u.a.string,id:u.a.string,label:u.a.string,position:u.a.oneOf(["top","left","bottom","right"])}},,,function(o,c,e){"use strict";e.r(c),e.d(c,"Grid",function(){return L}),e.d(c,"GridItem",function(){return T});var a,i=e(6),f=e.n(i),l=e(5),t=e.n(l),g=e(3),b=e.n(g),h=e(1),s=e.n(h),p=e(2),n=e.n(p),r=e(0),u=e.n(r),E=Object(h.createContext)({gap:0,gridCols:12}),z=e(4),A=e(7),F=["gap","gridCols"],O=n()(z.Box)(a||(a=b()([`
  display: grid;
  grid-template-columns: repeat(`,`, 1fr);
  `,`
`])),function(d){return d.gridCols},function(d){var C=d.theme,M=d.gap;return Object(A.a)("gap",M,C)}),L=function(d){var C=d.gap,M=d.gridCols,j=t()(d,F);return s.a.createElement(E.Provider,{value:{gap:C,gridCols:M}},s.a.createElement(O,f()({gap:C,gridCols:M},j)))};L.defaultProps={gap:0,gridCols:12},L.propTypes={gap:u.a.oneOfType([u.a.number,u.a.arrayOf(u.a.number)]),gridCols:u.a.number};var I,k=["col","xs","s"],m=n.a.div(I||(I=b()([`
  grid-column: span `,`;
  max-width: 100%;

  `,` {
    grid-column: span `,`;
  }

  `,` {
    grid-column: span `,`;
  }
`])),function(d){return d.col},function(d){return d.theme.mediaQueries.tablet},function(d){return d.s},function(d){return d.theme.mediaQueries.mobile},function(d){return d.xs}),T=function(d){var C=d.col,M=d.xs,j=d.s,U=t()(d,k),B=Object(h.useContext)(E),D=B.gap,w=B.gridCols;return s.a.createElement(m,{gap:D,gridCols:w,col:C,xs:M,s:j},s.a.createElement(z.Box,U))};T.defaultProps={col:void 0,s:void 0,xs:void 0},T.propTypes={col:u.a.number,s:u.a.number,xs:u.a.number}},,,function(o,c,e){"use strict";e.r(c),e.d(c,"Badge",function(){return r});var a=e(6),i=e.n(a),f=e(5),l=e.n(f),t=e(1),g=e.n(t),b=e(0),h=e.n(b),s=e(9),p=e(8),n=["active","textColor","backgroundColor","children","minWidth"],r=function(u){var E=u.active,z=u.textColor,A=u.backgroundColor,F=u.children,O=u.minWidth,L=l()(u,n);return g.a.createElement(s.Flex,i()({inline:!0,alignItem:"center",justifyContent:"center",minWidth:O,padding:1,background:E?"primary100":A,hasRadius:!0},L),g.a.createElement(p.Typography,{variant:"sigma",textColor:E?"primary600":z},F))};r.defaultProps={active:!1,backgroundColor:"neutral100",minWidth:5,textColor:"neutral600"},r.propTypes={active:h.a.bool,backgroundColor:h.a.string,children:h.a.oneOfType([h.a.number,h.a.string]).isRequired,minWidth:h.a.number,textColor:h.a.string}},,function(o,c,e){"use strict";e.r(c),e.d(c,"Divider",function(){return A});var a,i=e(6),f=e.n(i),l=e(5),t=e.n(l),g=e(3),b=e.n(g),h=e(1),s=e.n(h),p=e(0),n=e.n(p),r=e(4),u=e(2),E=["unsetMargin"],z=e.n(u)()(r.Box)(a||(a=b()([`
  height: 1px;
  border: none;
  `,`
`])),function(F){return F.unsetMargin?"margin: 0;":""}),A=function(F){var O=F.unsetMargin,L=t()(F,E);return s.a.createElement(z,f()({},L,{as:"hr",unsetMargin:O}))};A.defaultProps={background:"neutral150",unsetMargin:!0},A.propTypes={background:n.a.string,unsetMargin:n.a.bool}},,,,,,,,,function(o,c,e){"use strict";e.r(c),e.d(c,"IconButtonGroup",function(){return I}),e.d(c,"IconButton",function(){return k});var a,i,f=e(6),l=e.n(f),t=e(5),g=e.n(t),b=e(3),h=e.n(b),s=e(1),p=e.n(s),n=e(0),r=e.n(n),u=e(2),E=e.n(u),z=e(46),A=e(39),F=e(9),O=["label","noBorder","icon","disabled","onClick"],L=E()(A.BaseButton)(a||(a=h()([`
  display: flex;
  align-items: center;
  justify-content: center;
  height: `,`rem;
  width: `,`rem;

  svg {
    > g,
    path {
      fill: `,`;
    }
  }
  &:hover {
    svg {
      > g,
      path {
        fill: `,`;
      }
    }
  }
  &:active {
    svg {
      > g,
      path {
        fill: `,`;
      }
    }
  }
  &[aria-disabled='true'] {
    background-color: `,`;
    svg {
      path {
        fill: `,`;
      }
    }
  }
  `,`
`])),2,2,function(m){return m.theme.colors.neutral500},function(m){return m.theme.colors.neutral600},function(m){return m.theme.colors.neutral400},function(m){return m.theme.colors.neutral150},function(m){return m.theme.colors.neutral600},function(m){return m.noBorder?"border: none;":void 0}),I=E()(F.Flex)(i||(i=h()([`
  & span:first-child button {
    border-left: 1px solid `,`;
    border-radius: `,`;
  }

  & span:last-child button {
    border-radius: `,`;
  }

  & `,` {
    border-radius: 0;
    border-left: none;

    svg {
      path {
        fill: `,`;
      }
    }

    &:hover {
      background-color: `,`;

      svg {
        path {
          fill: `,`;
        }
      }
    }

    &:active {
      background-color: `,`;
      svg {
        path {
          fill: `,`;
        }
      }
    }

    &[aria-disabled='true'] {
      svg {
        path {
          fill: `,`;
        }
      }
    }
  }
`])),function(m){return m.theme.colors.neutral200},function(m){var T=m.theme;return"".concat(T.borderRadius," 0 0 ").concat(T.borderRadius)},function(m){var T=m.theme;return"0 ".concat(T.borderRadius," ").concat(T.borderRadius," 0")},L,function(m){return m.theme.colors.neutral700},function(m){return m.theme.colors.neutral100},function(m){return m.theme.colors.neutral800},function(m){return m.theme.colors.neutral150},function(m){return m.theme.colors.neutral900},function(m){return m.theme.colors.neutral600}),k=p.a.forwardRef(function(m,T){var d=m.label,C=m.noBorder,M=m.icon,j=m.disabled,U=m.onClick,B=g()(m,O),D=function(w){!j&&U&&U(w)};return d?p.a.createElement(z.Tooltip,{label:d},p.a.createElement(L,l()({},B,{ref:T,noBorder:C,onClick:D,"aria-disabled":j}),M)):p.a.createElement(L,l()({},B,{ref:T,noBorder:C,onClick:D,"aria-disabled":j}),M)});k.displayName="IconButton",k.defaultProps={label:void 0,noBorder:!1,disabled:!1,onClick:void 0},k.propTypes={disabled:r.a.bool,icon:r.a.element.isRequired,label:r.a.string,noBorder:r.a.bool,onClick:r.a.func}},,,,,,,,function(o,c){o.exports=ie},,,,function(o,c,e){"use strict";e.r(c),e.d(c,"Searchbar",function(){return B}),e.d(c,"SearchForm",function(){return D});var a,i,f,l=e(6),t=e.n(l),g=e(5),b=e.n(g),h=e(3),s=e.n(h),p=e(1),n=e.n(p),r=e(0),u=e.n(r),E=e(2),z=e.n(E),A=e(71),F=e.n(A),O=e(34),L=e.n(O),I=e(27),k=e(16),m=e(21),T=e(14),d=e(9),C=["name","size","children","value","onClear","clearLabel"],M=z()(d.Flex)(a||(a=s()([`
  font-size: 0.5rem;
  svg path {
    fill: `,`;
  }
`])),function(w){return w.theme.colors.neutral400}),j=z()(d.Flex)(i||(i=s()([`
  font-size: 0.8rem;

  svg path {
    fill: `,`;
  }
`])),function(w){return w.theme.colors.neutral800}),U=z.a.div(f||(f=s()([`
  border-radius: `,`;
  box-shadow: `,`;

  &:focus-within {
    `,` {
      svg path {
        fill: `,`;
      }
    }
  }

  `,` {
    border: 1px solid transparent;
  }

  `,`
`])),function(w){return w.theme.borderRadius},function(w){return w.theme.shadows.filterShadow},j,function(w){return w.theme.colors.primary600},k.InputWrapper,Object(T.c)(k.InputWrapper)),B=Object(p.forwardRef)(function(w,y){var P=w.name,S=w.size,x=w.children,N=w.value,q=w.onClear,G=w.clearLabel,Z=b()(w,C),ue=Object(p.useRef)(null),Ee=N.length>0,se=y||ue;return n.a.createElement(U,null,n.a.createElement(k.Field,{name:P},n.a.createElement(m.VisuallyHidden,null,n.a.createElement(k.FieldLabel,null,x)),n.a.createElement(k.FieldInput,t()({ref:se,value:N,startAction:n.a.createElement(j,null,n.a.createElement(F.a,{"aria-hidden":!0})),size:S,endAction:Ee?n.a.createElement(k.FieldAction,{label:G,onClick:function(Ce){q(Ce),ue.current.focus()}},n.a.createElement(M,null,n.a.createElement(L.a,null))):void 0},Z))))});B.displayName="Searchbar",B.defaultProps={value:"",size:"M"},B.propTypes={children:u.a.string.isRequired,clearLabel:u.a.string.isRequired,name:u.a.string.isRequired,onClear:u.a.func.isRequired,size:u.a.oneOf(Object.keys(I.a.input)),value:u.a.string};var D=function(w){return n.a.createElement("form",t()({},w,{role:"search"}))}},,,,,,,,,,,,,function(o,c){o.exports=le},,,,,,,function(o,c,e){"use strict";e.r(c),e.d(c,"subNavWidth",function(){return A}),e.d(c,"SubNav",function(){return O}),e.d(c,"SubNavHeader",function(){return P}),e.d(c,"SubNavLink",function(){return be}),e.d(c,"SubNavLinkSection",function(){return Re}),e.d(c,"SubNavSection",function(){return ke}),e.d(c,"SubNavSections",function(){return Ie});var a,i=e(6),f=e.n(i),l=e(5),t=e.n(l),g=e(3),b=e.n(g),h=e(1),s=e.n(h),p=e(2),n=e.n(p),r=e(0),u=e.n(r),E=e(49),z=["ariaLabel"],A="".concat(14.5,"rem"),F=n()(E.Grid)(a||(a=b()([`
  width: `,`;
  background: `,`;
  position: sticky;
  top: 0;
  height: 100vh;
  overflow-y: auto;
  border-right: 1px solid `,`;
  z-index: 1;
`])),A,function(v){return v.theme.colors.neutral100},function(v){return v.theme.colors.neutral200}),O=function(v){var _=v.ariaLabel,Y=t()(v,z);return s.a.createElement(F,f()({"aria-label":_,as:"nav"},Y))};O.propTypes={ariaLabel:u.a.string.isRequired};var L,I=e(10),k=e.n(I),m=e(71),T=e.n(m),d=e(9),C=e(8),M=e(63),j=e(4),U=e(54),B=e(75),D=e(15),w=e(11),y=n()(U.Divider)(L||(L=b()([`
  width: `,`rem;
  background-color: `,`;
`])),1.5,function(v){return v.theme.colors.neutral200}),P=function(v){var _=v.as,Y=v.label,K=v.searchLabel,re=v.searchable,ce=v.onChange,pe=v.value,ve=v.onClear,he=v.onSubmit,ge=v.id,xe=Object(h.useState)(!1),ye=k()(xe,2),me=ye[0],Pe=ye[1],Ye=function(ae){var Oe=Object(h.useRef)();return Object(h.useEffect)(function(){Oe.current=ae}),Oe.current}(me),Qe=Object(D.a)("subnav-searchbar-clear",ge),Se=Object(h.useRef)(),ze=Object(h.useRef)();return Object(h.useEffect)(function(){me&&Se.current&&Se.current.focus(),Ye&&!me&&ze.current&&ze.current.focus()},[me]),me?s.a.createElement(j.Box,{paddingLeft:4,paddingTop:5,paddingBottom:2,paddingRight:4},s.a.createElement(B.SearchForm,null,s.a.createElement(B.Searchbar,{name:"searchbar",value:pe,onChange:ce,placeholder:"e.g: strapi-plugin-abcd",onKeyDown:function(ae){ae.key===w.a.ESCAPE&&Pe(!1)},ref:Se,onBlur:function(ae){var Oe;((Oe=ae.relatedTarget)===null||Oe===void 0?void 0:Oe.id)!==Qe&&Pe(!1)},onClear:function(ae){ve(ae),Se.current.focus()},onSubmit:he,clearLabel:"Clear",size:"S"},K)),s.a.createElement(j.Box,{paddingLeft:2,paddingTop:4},s.a.createElement(y,null))):s.a.createElement(j.Box,{paddingLeft:6,paddingTop:6,paddingBottom:2,paddingRight:4},s.a.createElement(d.Flex,{justifyContent:"space-between",alignItems:"flex-start"},s.a.createElement(C.Typography,{variant:"beta",as:_},Y),re&&s.a.createElement(M.IconButton,{ref:ze,onClick:function(){Pe(function(ae){return!ae})},label:K,icon:s.a.createElement(T.a,null)})),s.a.createElement(j.Box,{paddingTop:4},s.a.createElement(y,null)))};P.defaultProps={as:"h2",searchable:!1,onChange:function(){},onClear:function(){},onSubmit:function(){},value:"",searchLabel:"",id:void 0},P.propTypes={as:u.a.string,id:u.a.string,label:u.a.string.isRequired,onChange:u.a.func,onClear:u.a.func,onSubmit:u.a.func,searchLabel:u.a.string,searchable:u.a.bool,value:u.a.string};var S,x,N,q=e(88),G=e.n(q),Z=e(36),ue=["children","icon","withBullet"],Ee=n()(j.Box)(S||(S=b()([`
  display: flex;
  align-items: center;
  justify-content: space-between;
  text-decoration: none;
  color: `,`;
  svg > * {
    fill: `,`;
  }

  &.active {
    `,`
  }

  &:focus-visible {
    outline-offset: -2px;
  }
`])),function(v){return v.theme.colors.neutral800},function(v){return v.theme.colors.neutral600},function(v){var _=v.theme;return`
      background-color: `.concat(_.colors.primary100,`;
      border-right: 2px solid `).concat(_.colors.primary600,`;
      svg > * {
        fill: `).concat(_.colors.primary700,`;
      }
      `).concat(C.Typography,` {
        color: `).concat(_.colors.primary700,`;
        font-weight: 500;
      }
      `)}),se=n()(G.a)(x||(x=b()([`
  width: `,`rem;
  height: `,`rem;
  * {
    fill: `,`;
  }
`])),.75,.25,function(v){var _=v.theme;return v.$active?_.colors.primary600:_.colors.neutral600}),Ce=n.a.div(N||(N=b()([`
  svg {
    height: `,`rem;
    width: `,`rem;
  }
`])),.75,.75),be=function(v){var _=v.children,Y=v.icon,K=v.withBullet,re=t()(v,ue);return s.a.createElement(Ee,f()({as:Z.NavLink,icon:Y,background:"neutral100",paddingLeft:7,paddingBottom:2,paddingTop:2},re),s.a.createElement(d.Flex,null,Y?s.a.createElement(Ce,null,Y):s.a.createElement(se,null),s.a.createElement(j.Box,{paddingLeft:2},s.a.createElement(C.Typography,{as:"span"},_))),K&&s.a.createElement(j.Box,{as:d.Flex,paddingRight:4},s.a.createElement(se,{$active:!0})))};be.displayName="SubNavLink",be.defaultProps={icon:null,withBullet:!1},be.propTypes={children:u.a.node,icon:u.a.element,link:u.a.element,withBullet:u.a.bool};var Te,ee,ne,V=e(35),$=e.n(V),oe=n()(j.Box)(Te||(Te=b()([`
  max-height: `,`rem;
  svg {
    height: `,`rem;
    path {
      fill: `,`;
    }
  }
`])),2,.25,function(v){return v.theme.colors.neutral700}),de=n.a.button(ee||(ee=b()([`
  border: none;
  padding: 0;
  background: transparent;
  display: flex;
  align-items: center;
`]))),fe=n.a.div(ne||(ne=b()([`
  display: flex;
  align-items: center;
  width: `,`rem;
  transform: rotateX(`,`);
`])),.75,function(v){return v.rotated?"0deg":"180deg"}),Re=function(v){var _=v.label,Y=v.children,K=v.id,re=Object(h.useState)(!0),ce=k()(re,2),pe=ce[0],ve=ce[1],he=Object(D.a)("subnav-list",K);return s.a.createElement(j.Box,null,s.a.createElement(oe,{paddingLeft:7,paddingTop:2,paddingBottom:2,paddingRight:4},s.a.createElement(d.Flex,{justifyContent:"space-between"},s.a.createElement(de,{onClick:function(){ve(function(ge){return!ge})},"aria-expanded":pe,"aria-controls":he},s.a.createElement(fe,{rotated:pe},s.a.createElement($.a,{"aria-hidden":!0})),s.a.createElement(j.Box,{paddingLeft:2},s.a.createElement(C.Typography,{as:"span",fontWeight:"semiBold",textColor:"neutral800"},_))))),pe&&s.a.createElement("ul",{id:he},h.Children.map(Y,function(ge,xe){return s.a.createElement("li",{key:xe},ge)})))};Re.defaultProps={id:void 0},Re.propTypes={children:u.a.node,id:u.a.string,label:u.a.string.isRequired};var Le,Fe,Me,We,He=e(52),_e=n()(d.Flex)(Le||(Le=b()([`
  border: none;
  padding: 0;
  background: transparent;
`]))),Ne=n.a.div(Fe||(Fe=b()([`
  display: flex;
  align-items: center;
  transform: rotateX(`,`);
`])),function(v){return v.rotated?"0deg":"180deg"}),Be=function(v){var _=v.collapsable,Y=v.label,K=v.onClick,re=v.ariaExpanded,ce=v.ariaControls;return _?s.a.createElement(_e,{as:"button",onClick:K,"aria-expanded":re,"aria-controls":ce},s.a.createElement(j.Box,{paddingRight:1},s.a.createElement(C.Typography,{variant:"sigma",textColor:"neutral600"},Y)),_&&s.a.createElement(Ne,{rotated:re},s.a.createElement($.a,{"aria-hidden":!0}))):s.a.createElement(_e,null,s.a.createElement(j.Box,{paddingRight:1},s.a.createElement(C.Typography,{variant:"sigma",textColor:"neutral600"},Y)))};Be.defaultProps={ariaControls:null,ariaExpanded:null,collapsable:!1,onClick:function(){}},Be.propTypes={ariaControls:u.a.string,ariaExpanded:u.a.bool,collapsable:u.a.bool,label:u.a.string.isRequired,onClick:u.a.func};var De=n()(j.Box)(Me||(Me=b()([`
  max-height: `,`rem;
  svg {
    height: `,`rem;
    path {
      fill: `,`;
    }
  }
`])),2,.25,function(v){return v.theme.colors.neutral500}),qe=n()(He.Badge)(We||(We=b()([`
  display: flex;
  align-items: center;
`]))),ke=function(v){var _=v.collapsable,Y=v.label,K=v.badgeLabel,re=v.children,ce=v.id,pe=Object(h.useState)(!0),ve=k()(pe,2),he=ve[0],ge=ve[1],xe=Object(D.a)("subnav-list",ce);return s.a.createElement(j.Box,null,s.a.createElement(De,{paddingLeft:6,paddingTop:2,paddingBottom:2,paddingRight:4},s.a.createElement(d.Flex,{justifyContent:"space-between"},s.a.createElement(Be,{onClick:function(){ge(function(ye){return!ye})},ariaExpanded:he,ariaControls:xe,collapsable:_,label:Y}),K&&s.a.createElement(qe,{backgroundColor:"neutral150",textColor:"neutral600"},K))),(!_||he)&&s.a.createElement("ul",{id:xe},h.Children.map(re,function(ye,me){return s.a.createElement("li",{key:me},ye)})))};ke.defaultProps={badgeLabel:null,collapsable:!1,id:void 0},ke.propTypes={badgeLabel:u.a.string,children:u.a.node,collapsable:u.a.bool,id:u.a.string,label:u.a.string.isRequired};var Ue=e(26),Ge=["children"],Ie=function(v){var _=v.children,Y=t()(v,Ge);return s.a.createElement(j.Box,{paddingTop:2,paddingBottom:4},s.a.createElement(Ue.Stack,f()({as:"ul",spacing:2},Y),h.Children.map(_,function(K,re){return s.a.createElement("li",{key:re},K)})))};Ie.propTypes={children:u.a.node.isRequired}}])})},57235:function(we,Ae,Q){(function(je,te){we.exports=te(Q(32735))})(this,function(je){return function(te){var J={};function H(R){if(J[R])return J[R].exports;var W=J[R]={i:R,l:!1,exports:{}};return te[R].call(W.exports,W,W.exports,H),W.l=!0,W.exports}return H.m=te,H.c=J,H.d=function(R,W,X){H.o(R,W)||Object.defineProperty(R,W,{enumerable:!0,get:X})},H.r=function(R){typeof Symbol!="undefined"&&Symbol.toStringTag&&Object.defineProperty(R,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(R,"__esModule",{value:!0})},H.t=function(R,W){if(1&W&&(R=H(R)),8&W||4&W&&typeof R=="object"&&R&&R.__esModule)return R;var X=Object.create(null);if(H.r(X),Object.defineProperty(X,"default",{enumerable:!0,value:R}),2&W&&typeof R!="string")for(var ie in R)H.d(X,ie,function(le){return R[le]}.bind(null,ie));return X},H.n=function(R){var W=R&&R.__esModule?function(){return R.default}:function(){return R};return H.d(W,"a",W),W},H.o=function(R,W){return Object.prototype.hasOwnProperty.call(R,W)},H.p="",H(H.s=53)}({0:function(te,J){te.exports=je},53:function(te,J,H){"use strict";H.r(J);var R=H(0);function W(){return(W=Object.assign||function(X){for(var ie=1;ie<arguments.length;ie++){var le=arguments[ie];for(var o in le)Object.prototype.hasOwnProperty.call(le,o)&&(X[o]=le[o])}return X}).apply(this,arguments)}J.default=function(X){return R.createElement("svg",W({width:"1em",height:"1em",viewBox:"0 0 4 4",fill:"none",xmlns:"http://www.w3.org/2000/svg"},X),R.createElement("rect",{width:4,height:4,rx:2,fill:"#A5A5BA"}))}}})})}}]);
