// @mantine/notifications/esm/events.js
import { useEffect } from 'react';

const NOTIFICATIONS_EVENTS = {
  show: "mantine:show-notification",
  hide: "mantine:hide-notification",
  update: "mantine:update-notification",
  clean: "mantine:clean-notifications",
  cleanQueue: "mantine:clean-notifications-queue"
};
function createEvent(type, detail) {
  return new CustomEvent(type, { detail });
}
function showNotification(notification) {
  window.dispatchEvent(createEvent(NOTIFICATIONS_EVENTS.show, notification));
}
function updateNotification(notification) {
  window.dispatchEvent(createEvent(NOTIFICATIONS_EVENTS.update, notification));
}
function hideNotification(id) {
  window.dispatchEvent(createEvent(NOTIFICATIONS_EVENTS.hide, id));
}
function cleanNotifications() {
  window.dispatchEvent(createEvent(NOTIFICATIONS_EVENTS.clean));
}
function cleanNotificationsQueue() {
  window.dispatchEvent(createEvent(NOTIFICATIONS_EVENTS.cleanQueue));
}
function useNotificationsEvents(ctx) {
  const events = {
    show: (event) => ctx.showNotification(event.detail),
    hide: (event) => ctx.hideNotification(event.detail),
    update: (event) => ctx.updateNotification(event.detail.id, event.detail),
    clean: ctx.clean,
    cleanQueue: ctx.cleanQueue
  };
  useEffect(() => {
    Object.keys(events).forEach((event) => {
      window.addEventListener(NOTIFICATIONS_EVENTS[event], events[event]);
    });
    return () => {
      Object.keys(events).forEach((event) => {
        window.removeEventListener(NOTIFICATIONS_EVENTS[event], events[event]);
      });
    };
  }, []);
}

export { NOTIFICATIONS_EVENTS, cleanNotifications, cleanNotificationsQueue, createEvent, hideNotification, showNotification, updateNotification, useNotificationsEvents };
